harmoni_core/harmoni_recorder/src/harmoni_recorder/template_storedata.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+# Common Imports
+import rospy
+import roslib
+from mongodb_client import MongoDBClient
+
+
+def main():
+    try:
+        name="mongodb_client"
+        rospy.init_node(name)
+        mdb_client = MongoDBClient(client_uri="mongodb://172.18.3.3:27017/", username="root", password="example")
```

Comment:
Agreed. What do you think about creating a single .env file for all the secret credentials instead of many of them (one for each service)?
Maybe we can open an issue and resolve it later.

Reply:
I think the problem with joining them is specific services expect credentials in a specific form. Not sure you can get around having multiple secret files, but we could have a single source of truth for their locations.